lava_test_plans/projects/benchmarking/devices/bcm2711-rpi-4-b

```diff
@@ -0,0 +1,18 @@
+{% extends "devices/bcm2711-rpi-4-b" %}
```

Comment:
Can we just add the mmtests bits parts into a patch and push that into its own PR?
"
testcases: add mmtests subsuite sysbench-cpu

Creating a generic template-mmtests.jinja2 file that can be reused
when running other suites inside the mmtests repository.
For now start to add mmtests-sysbench-cpu as a testcases.

Signed-off-by: ...
"